### PR TITLE
Script to set up testnet development

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

Running development against testnet requires several steps:
You need
1. a canister_ids.json file
2. several tools installed
3. to run `DFX_NETWORK=$network ./config.sh`
4. to run `npm ci` and `npm run dev` from `frontend/`

It's easy to forget something, especially if you're new to the team.
This scripts checks all the inputs and runs all the steps so you can't forget something.

# Changes

1. Add `dev-testnet.sh` analogous to `dev-local.sh`.
2. The script needs to be run from a git repo and will work from any directory inside the repo so you don't need to `cd` into or out of `frontend/`.
3. Change to `dev-local.sh` to make the previous point work for it as well.

# Tests

Tested manually with a new user account on my Mac.

### CODE REVIEW

File: scripts/dev-testnet.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+need node
+
+networks=$(jq -r '[.[]|keys[]]|unique|join(" ")' canister_ids.json)
+
+usage() {
+  echo "Usage: $0 <network>"
+  echo "Available networks are: $networks"
+}
+
+if [ "$#" != "1" ]; then
```
Comment (lmuntaner) [inline scripts/dev-testnet.sh:52]: I'm not very familiar with bash, what does this do?

**Review by lmuntaner — APPROVED** [review]
Just a question, but already approving. LGTM, thanks!

File: scripts/dev-testnet.sh (hunk 1 above)
Comment (dskloetd) [reply scripts/dev-testnet.sh:52]: `$#` is the number of arguments passed to the script.
This checks if the number of arguments is not 1 and then it outputs usage instructions and exits with an error code.

Comment (lmuntaner) [reply scripts/dev-testnet.sh:52]: Thanks for the explanation!

**Pull request merged**